coordinator/src/main/java/org/apache/uniffle/coordinator/web/resource/ServerResource.java

```diff
@@ -69,19 +71,13 @@ public Response<List<ServerNode>> nodes(@QueryParam("status") String status) {
       serverList = clusterManager.getUnhealthyServerList();
     } else if (ServerStatus.LOST.name().equalsIgnoreCase(status)) {
       serverList = clusterManager.getLostServerList();
+    } else if (ServerStatus.DECOMMISSIONED.name().equalsIgnoreCase(status)) {
+      serverList = clusterManager.getDecommissionedServerList();
+    } else if (ServerStatus.DECOMMISSIONING.name().equalsIgnoreCase(status)) {
+      serverList = clusterManager.getDecommissioningServerList();
```

Comment:
We can remove L74 to L77 because of #1174 .

Reply:
> We can remove L74 to L77 because of #1174 .

If this is removed, how can the two DECOMMISSIONED and DECOMMISSIONING servernodes be obtained?

Reply:
You can take a look at the original logic in the master branch.
https://github.com/apache/incubator-uniffle/blob/f793a22dc8e5b821611b662e2a3227faac0731a2/coordinator/src/main/java/org/apache/uniffle/coordinator/web/resource/ServerResource.java#L75-L84